actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -290,12 +291,10 @@ pub fn callactor<BS: Blockstore, RT: Runtime<BS>>(
         .map_err(|_| StatusCode::InvalidMemoryAccess)?;
 
     let result = {
-        let dst_addr: EthAddress = dst.try_into()?;
+        // TODO: this is wrong https://github.com/filecoin-project/ref-fvm/issues/1018
+        let dst_addr: EthAddress = dst.into();
         let dst_addr: Address = dst_addr.try_into()?;
 
-        if method.bits() > 64 {
```

Comment:
as_u64 can panic if it doesnt fit, so i think we should keep this check.

Reply:
Ah, I see. I switched to `low_u64` to match the rest of the EVM.

Reply:
Hm. Actually, no, I think we'd be better off by returning an error code. SGTY?

Reply:
yeah, debuggability.

Reply:
Ok, I'm just leaving this as-is for now. We can fix it later.

Reply:
(i.e., leaving it as you wrote it, which is probably more correct than what I was doing anyways...)